imageio/plugins/gdal.py

```diff
@@ -40,7 +40,7 @@ class GdalFormat(Format):
     def _can_read(self, request):
         if request.extension in (".ecw",):
             return True
-        if has_module("osgeo.gdal"):
+        if has_module("gdal", "osgeo"):
```

Comment:
I changed `has_module` in a different branch, and it can now be used as before:
```suggestion
        has_module("osgeo.gdal"):
```

Reply:
Nice, thanks @almarklein

Reply:
Also see the other uses of has_module ...